docs/k8s/guides/custom-domain.mdx

```diff
@@ -3,11 +3,27 @@ sidebar_label: Custom Domains
 title: Kubernetes Custom White Label Domains
 ---
 
-In the Kubernetes Ingress and Gateway API specs, ingresses and routes can have multiple rules with different hostnames. While standard ngrok domains are available for use immediately after reservation, custom white label domains may require a couple extra steps to get working. The following outlines 2 options for getting custom white label domains working with the ngrok Kubernetes Operator.
+In the Kubernetes Ingress and Gateway API specs, ingresses and routes can have multiple rules with different hostnames. While standard ngrok
+domains are available for use immediately after reservation, custom white label domains may require a couple extra steps to get working.
+The following outlines 2 options for getting custom white label domains working with the ngrok Kubernetes Operator.
```

Comment:
Is it technically three now?

```suggestion
The following outlines three options for getting custom white label domains working with the ngrok Kubernetes Operator.
```

Reply:
I think it is still 2 options at a high level(managed vs unmanaged). Open to other input & suggestions though!

Reply:
Fair enough! I won't hold you to that!